templates/service-template.yml

```
@@ -595,6 +605,15 @@ objects:
     data:
       deny-list-configuration.yaml: |-
         ${DENIED_USERS}
+  - kind: ConfigMap
+    apiVersion: v1
+    metadata:
+      name: kas-fleet-manager-access-organisations-config
```

```suggestion
      name: kas-fleet-manager-accepted-organisations-config
```

I'd suggest to change the name to `kas-fleet-manager-accepted-organisations-config` throughout